testsuite/MDAnalysisTests/__init__.py

```diff
@@ -87,14 +87,14 @@
 The unittests use the :mod:`unittest` module together with nose_. See the
```

Comment:
Add an entry to `testsuite/CHANGELOG` – this is a noteworthy change in the tests!!